types/staking.go

```diff
@@ -2,17 +2,27 @@ package types
 
 type ProposalDetail struct {
 	ProposalMetadata
-	VoteYes     uint64            `json:"voteYes"`
-	VoteNo      uint64            `json:"voteNo"`
-	VoteAbstain uint64            `json:"voteAbstain"`
-	Params      map[string]string `json:"params"`
+	VoteYes             uint64                   `json:"voteYes" bson:"voteYes"`
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `gofmt`-ed with `-s` (gofmt)